ambry-named-mysql/src/test/java/com/github/ambry/named/MySqlNamedBlobDbTest.java

```diff
@@ -126,6 +181,62 @@ private void triggerQueryExecutionError(String datacenter, SQLException executio
         throw new RuntimeException(e);
       }
     }
+
+    private void triggerQueryExecutionErrorForLocalDataCenter(List<String> datacenters) {
```

Comment:
maybe call these two `triggerEmptyResultSetForLocalDatacenter` and `triggerEmptyResultSetForAllDatacenters`.

Reply:
Updated.